persistence-utils-demo-webtest/pom.xml

```diff
@@ -7,6 +7,7 @@
   <packaging>iar-integration-test</packaging>
   <description>Web Test project of the persistence utils demo project.</description>
   <properties>
+    <project.build.plugin.version>11.3.0</project.build.plugin.version>
     <webtester.version>11.3.0</webtester.version>
```

Comment:
use the`tester.version` property, so that the 'dev-pipeline' may override this value